app/components/action-feed/component.js

```diff
@@ -12,15 +12,54 @@ import { translationMacro as t } from 'ember-i18n';
  * ```
  * @class action-feed
  */
+
 export default Component.extend({
     store: service(),
     toast: service(),
-
+    currentUser: service(),
     classNames: ['action-feed'],
+    page: 1,
+    listLoading: false,
 
     errorMessage: t('components.action-feed.error_loading'),
 
     moreActions: computed('totalPages', 'page', function() {
         return this.get('page') < this.get('totalPages');
     }),
+
+    init() {
+        this._super(...arguments);
+        this.actionsList = [];
+        this.loadPage();
+        this.set('listLoading', true);
```

Comment:
Seems like `listLoading` and `loadingPage` could be combined -- check whether `loadingPage` is truthy in the template and reset it to `null` when it's done loading.